packages/benchmarks/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+pub mod fibonacci;
+
+use std::fs;
+use std::path::Path;
+use std::sync::{Arc};
+use polywrap_core::file_reader::SimpleFileReader;
+use polywrap_tests_utils::helpers::get_tests_path;
+use polywrap_wasm::wasm_wrapper::WasmWrapper;
+
+pub fn get_fibonacci_dir(implementation: &str) -> String {
+    let relative_path = format!("fibonacci/{}/build", implementation);
+    Path::new(&relative_path)
+        .canonicalize()
+        .unwrap()
+        .into_os_string()
+        .into_string()
+        .unwrap()
+}
+
+pub fn get_fibonacci_wrap(implementation: &str) -> WasmWrapper {
+    let path = Path::new(&get_fibonacci_dir(implementation)).join("wrap.wasm");
+    let module_bytes = fs::read(path).unwrap();
```

Comment:
You can use include_bytes! for this instead of the file system to get some performence increase. Check out how it's done in the default-config crate.
Note: you wont be able to pass in the implementation str, but that's an ok compromise IMO

Reply:
That's awesome. This isn't affecting the benchmarks. Do you think the change still needs to be made?

Reply:
I think so, if it's not too much trouble. It makes sense cause it decreases time it takes for the tests to run (so developers can confirm their code works faster). Ideally, we'd use include_bytes everywhere instead of fs::read everywhere. Ofc, one-off uses don't affect much :D